tests/ci/upload_result_helper.py

```diff
@@ -1,41 +1,43 @@
+from pathlib import Path
+from typing import Dict, List
 import os
 import logging
-import ast
 
 from env_helper import (
     GITHUB_JOB_URL,
     GITHUB_REPOSITORY,
     GITHUB_RUN_URL,
     GITHUB_SERVER_URL,
 )
-from report import ReportColorTheme, create_test_html_report
+from report import ReportColorTheme, TestResults, create_test_html_report
+from s3_helper import S3Helper
 
 
 def process_logs(
-    s3_client, additional_logs, s3_path_prefix, test_results, with_raw_logs
-):
+    s3_client: S3Helper,
+    additional_logs: List[str],
+    s3_path_prefix: str,
```

Comment:
This one is internal method? Check should run `upload_results`?

Reply:
Yes, nobody runs process_logs. It was the main place why I even started this refactoring. Nobody knew what `test_results` was in the past.